tests/dummy/app/pods/demo/controller.js

```diff
@@ -1,10 +1,36 @@
 import Ember from 'ember'
-
-export default Ember.Controller.extend({
-  dummyRecords: Ember.computed.alias('model'),
-
-  podsVisibility: Ember.A(),
-
+const {
+  Object,
+  Controller,
+  computed,
+  A
+} = Ember
+export default Controller.extend({
+  dummyRecords: computed.alias('model'),
+  orientation: 'vertical',
+  listItems: A([
+    Object.create({
+      title: 'Item',
+      description: 'Content'
+    }),
+    Object.create({
+      title: 'Item',
+      description: 'Content'
+    })
+  ]),
+  podsVisibility: A(),
+  detailActions: A([
```

Comment:
I think we should make this `buttons` since we'll have another set for `links` later